notebooks/official/matching_engine/two-tower-model-introduction.ipynb

```diff
@@ -0,0 +1,1269 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !gsutil ls {OUTPUT_DIR}
</pre><p>Add space after exclamation to be consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>